src/diffusers/modeling_utils.py

```diff
@@ -503,6 +504,100 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
 
         return model
 
+    @classmethod
+    def _get_model_file(
+        cls,
+        pretrained_model_name_or_path,
```

Comment:
@patrickvonplaten I made this a class method to access the class name more easily, lmk if it should stay a static method as in https://github.com/huggingface/diffusers/pull/1357

Reply:
ok for me